packages/react-next/src/components/Autofill/Autofill.types.ts

```diff
@@ -47,6 +47,13 @@ export interface IAutofill {
  * {@docCategory Autofill}
  */
 export interface IAutofillProps extends React.InputHTMLAttributes<HTMLInputElement | Autofill> {
+  /**
+   * The same as an Input's value property.
+   * While it can be any of these types, only a string should be passed in.
+   * The rest of the types (string[], number) are currently provided to keep this inline
+   * with the type provided by InputHTMLAttributes but may be removed at a later time
+   */
+  value?: string | string[] | number | undefined;
```

Comment:
Sorry to keep belaboring this, but I still don't think there's any reason to keep the rest of the types.

Reply:
We have no reasonable expectation that anyone should be using them (and especially with this being a major version, I'm fine with introducing a compiler error and making them fix it if they are), and including them just makes the definition unnecessarily confusing.

Reply:
I am going to remove this definition then so we can break it easily in a future release but I'm going to keep the native input types. I think that there is already going to be a fair amount of churn and this is a nice interim that doesn't break people much. It preserves the existing behavior since this isn't one of the components that we're refreshing.